### ISSUE TITLE
Epoch.time_slice() is crrently broken

### ISSUE THREAD

**Issue by grg2rsr** [NeuralEnsemble/python-neo#594]
currently, `Epoch.time_slice()` is broken:

```
import neo
import quantities as pq
Epoch = neo.core.Epoch(times=[1,2,3]*pq.s,durations=[1,1,1]*pq.ms,labels=['a','b','c'])
Epoch.time_slice(*(1,2)*pq.s)

## -- End pasted text --
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-d675ce8b57e8> in <module>()
      2 import quantities as pq
      3 Epoch = neo.core.Epoch(times=[1,2,3]*pq.s,durations=[1,1,1]*pq.ms,labels=['a','b','c'])
----> 4 Epoch.time_slice(*(1,2)*pq.s)

~/Dropbox/python/git_repos/python-neo/neo/core/epoch.py in time_slice(self, t_start, t_stop)
    261 
    262         indices = (self >= _t_start) & (self <= _t_stop)
--> 263         new_epc = self[indices]
    264         return new_epc
    265 

~/Dropbox/python/git_repos/python-neo/neo/core/epoch.py in __getitem__(self, i)
    177         obj._copy_data_complement(self)
    178         obj.durations = self.durations[i]
--> 179         obj.labels = self.labels[i]
    180         return obj
    181 

TypeError: only integer scalar arrays can be converted to a scalar index
```
`self.labels` is a list, and indexing it with a `np.array()` causes the error. I am making a pull request and reference this issue

**Comment by samuelgarcia** [comment]
Thanks for  the report.
My guess is that it is already fix in this PR #472
Could test the branch related to this PR  to see if the bug is fixed ?

Best

**Comment by grg2rsr** [comment]
my bad. All of this error boils down to me passing the labels as a list of strings and not a `np.array` of strings. Apologies ...
